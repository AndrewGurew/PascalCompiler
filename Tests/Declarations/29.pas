procedure bar(a:double; c:integer); forward;
function foo():integer; forward;