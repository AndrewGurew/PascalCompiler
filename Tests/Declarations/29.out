Declaration
 ⎬foo - Function
 Params
 Return type: Integer



 ⎬bar - Procedure
 Params: a - Double, c - Integer


